sdk/cognitiveservices/cognitiveservices-luis-runtime/README.md

```diff
@@ -145,4 +143,4 @@ See https://github.com/Azure/ms-rest-browserauth to learn how to authenticate to
 
 - [Microsoft Azure SDK for Javascript](https://github.com/Azure/azure-sdk-for-js)
 
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js/sdk/cognitiveservices/cognitiveservices-luis-runtime/README.png)
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js/sdk/README.png)
```

Comment:
Did this change come from the auto-generated code?

Reply:
It's auto-generated. all codes except in the README are auto-generated.

Reply:
@scbedd Looping you in here as you last worked on impressions. I believe this change is not what we want. Can you confirm?

Reply:
I have talked to @scbedd offline.
This needs to be fixed in the code generator, but meanwhile for now, please manually change this line to match what is in the master branch 

```suggestion
![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js/sdk/cognitiveservices/cognitiveservices-luis-runtime/README.png)
```